frontend/javascripts/oxalis/model/helpers/nml_helpers.ts

```diff
@@ -502,8 +502,11 @@ function serializeTreeGroups(treeGroups: Array<TreeGroup>, trees: Array<Tree>):
       serializeTagWithChildren(
         "group",
         {
-          id: treeGroup.groupId,
-          name: treeGroup.name,
+          ...{
+            id: treeGroup.groupId,
+            name: treeGroup.name,
+          },
```

Comment:
the original code should also work, unless I'm mistaken?

```suggestion
          id: treeGroup.groupId,
          name: treeGroup.name,
```

Reply:
no youre right, much better!